extra/models/inception.py

```diff
@@ -4,6 +4,8 @@
 from tinygrad.helpers import fetch
 
 from typing import Optional, Dict
+import numpy as np
+from scipy import linalg
```

Comment:
i think this is fine for now, but as tinygrad itself does not have these dependency, think more about if an import needs to be in the top level